app/assets/javascripts/models/beta/story.js

```diff
@@ -108,6 +108,16 @@ export const findById = (stories, id) => {
   return stories.find(story => story.id === id)
 }
 
+export const sort = async (story) => {
+  const newStory = serialize(story);
```

Comment:
`serialize` won't create a new story, it'll only create a new representation of the same story, we could make the name reflect that:

```suggestion
  const serializedStory = serialize(story);
```

Reply:
Done. Also fixed in existing methods